hack/update-generated.sh

```diff
@@ -3,7 +3,7 @@ STARTTIME=$(date +%s)
 source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
 # Update test names
-go generate -mod vendor ./test/extended
+ENABLE_STORAGE_GCE_PD_DRIVER=yes go generate -mod vendor ./test/extended
```

Comment:
include a comment/link about why this is here.

Reply:
Added comment back.